app/build.gradle

```diff
@@ -49,6 +53,8 @@ android {
 }
 
 dependencies {
+    // OneSignal SDK
+    implementation 'com.onesignal:OneSignal:[4.0.0, 4.99.99]'
```

Comment:
I think we can just specify an exact version and then update if needed

Reply:
and here we can specify `4.6.7` see https://github.com/OneSignal/OneSignal-Android-SDK/releases